pkgs/applications/misc/geoipupdate/default.nix

```diff
@@ -0,0 +1,22 @@
+{ stdenv, buildGoModule, fetchFromGitHub }:
+
+buildGoModule rec {
+  pname = "geoipupdate";
+  version = "4.1.5";
+
+  src = fetchFromGitHub {
+    owner = "maxmind";
+    repo = "geoipupdate";
+    rev = "v${version}";
+    sha256 = "1k0bmsqgw35sdmaafinlr4qd5910fi598i8irxrz11394d3c8giv";
+  };
+
+  modSha256 = "0mk6zp6byq3jc6wipx53bg5igry114klq5w8isc0z6r63zjsk6f6";
+
+  meta = with stdenv.lib; {
+    description = "Automatic GeoIP database updater";
+    license = with licenses; [ asl20 ];
+    platforms = platforms.all;
```

Comment:
This line is not needed:
https://github.com/NixOS/nixpkgs/blob/74edb58ad19225222a72c14bb37f3b42e5f0a344/pkgs/development/go-modules/generic/default.nix#L200-L201